pkg/plugins/resources/shell/condition_test.go

```diff
@@ -81,15 +86,17 @@ func TestShell_ConditionFromSCM(t *testing.T) {
 		wantErr       bool
 		wantCommand   string
 		commandResult commandResult
+		shell         string
 	}{
 		{
 			name:        "Successful Condition in existing SCM",
 			command:     "echo Hello",
+			shell:       "/bin/bash",
 			source:      "1.2.3",
 			scmDir:      "/dummy/dir",
 			wantResult:  true,
 			wantErr:     false,
-			wantCommand: "echo Hello 1.2.3",
+			wantCommand: "/bin/bash /tmp/updatecli/bin/9adca274a8e9298dd9060a3bd68649e74950b670b9bc4efe47443e1843091683.sh",
```

Comment:
This line is the cause of the latest failure: it assumes that the temporary directory is `/tmp` which is untrue on macOS, CentOS and other OSes.

The runtime code uses the golang's [`os.TempDir()`](https://github.com/updatecli/updatecli/pull/1067/files#diff-abe79d7eaac7a2d49a75891471c330f940a36b1d4434e8a6ddde65d9ab127803R10) which returns the value of `$TMPDIR` or `/tmp` if empty (ref. https://pkg.go.dev/os#TempDir).

You might want to capture this path at the beginning of the test and then adapt the line here, WDYT @olblak (should be easy)

Reply:
Thank you for testing, I'll have a look how.
I know wondering if I should also run the test by default in a windows runner to catch such issues